packages/api/src/controllers/access-control.ts

```diff
@@ -31,18 +31,35 @@ accessControl.post(
       (await db.stream.getByPlaybackId(playbackId)) ||
       (await db.asset.getByPlaybackId(playbackId));
 
+    let contentLog;
+    let userLog;
+    let signingKeyLog;
+
     if (!content || content.deleted) {
+      try {
+        contentLog = JSON.stringify(content);
+      } catch (e) {
+        contentLog = content;
+      }
```

Comment:
```suggestion
    const contentLog = JSON.stringify(content);
    const userLog = JSON.stringify(user);
    const signingKeyLog = JSON.stringify(signingKey);
```

There's no reasonable scenario where stringify will throw, and when it throws we don't want to catch it anyway (e.g. OOM), especially for an object that just came from our DB. So we can simplify this logic and avoid so many branches.

Reply:
While at it, we could also escape the log streams so that loki is able to parse the fields properly, WDYT? The easiest way that I know of is to stringify twice, which will encode the stringified string as a JSON string, i.e. escaped
```suggestion
    const contentLog = JSON.stringify(JSON.stringify(content));
    const userLog = JSON.stringify(JSON.stringify(user));
    const signingKeyLog = JSON.stringify(JSON.stringify(signingKey));
```

Reply:
Right, for some reason I thought in typescript JSON.stringify(undefined) would throw. Much better, gonna remove all the try catches!